beacon_node/src/cli.rs

```diff
@@ -296,6 +296,7 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 This is enabled by default, using default quotas. To disable rate limiting pass \
                 `disabled` to this option instead."
             )
+            .min_values(0)
```

Comment:
I don't think we need this, since it means we can pass `--inbound-rate-limiter` which would be equivalent to not passing the option at all

Reply:
Ohh yeah right. It needs a `takes_value` in that case. Sorry got confused with the previous option.

Reply:
you are right! good team work :tada: